### ISSUE TITLE
LBAS deployment fails if the node is covered by the initial LBAS UI. 

### ISSUE THREAD

**Issue by sakura-quest** [perryhuynh/kcauto#110]
#### Environment details

Version: 0.4.0
Browser: chrome (kc3)
OS: Windows 10

#### Issue variability
Is the issue consistent (can be reproduced consistently) or is it intermittent (only happens sometimes)?:
Yes

#### Issue

If the LBAS group UI is covering the node that you are trying to deploy to, the LBAS deployment fails and the program exits out. The ideal behavior should be that if it can't find the desired node upon sortie, the mouse should be moved over to the current location of the LBAS group UI. This should move the LBAS UI to the other side, after which the desired node should be searched for again. If it fails to find the node a  second time even after moving the LBAS UI to the other side, the program can exit out with an error. 

#### Log (if applicable)

The following log is from when trying to deploy LBAS to node B on e-5 of the Winter 2018 event. 
```
[94m[2018-03-10 11:28:29] Beginning combat sortie.[0m
[94m[2018-03-10 11:28:34] Assign LBAS groups to nodes.[0m
[94m[2018-03-10 11:28:34] Assigning nodes for LBAS group 2.[0m
[error] script [ E:\kcauto-kai\kcauto-kai.sikuli ] stopped with error at line --unknown--
[error] Error caused by: Traceback (most recent call last):
  File "E:\kcauto-kai\kcauto-kai.sikuli\kcauto-kai.py", line 66, in <module>
    kcauto_kai.run_combat_cycle()
  File "E:\kcauto-kai\kcauto-kai.sikuli\kcauto-kai.py", line 66, in <module>
    kcauto_kai.run_combat_cycle()
  File "E:\kcauto-kai\kcauto-kai.sikuli\main.py", line 317, in run_combat_cycle
    if self.modules['combat'].combat_logic_wrapper():
  File "E:\kcauto-kai\kcauto-kai.sikuli\combat.sikuli\combat.py", line 137, in combat_logic_wrapper
    self._run_combat_logic()
  File "E:\kcauto-kai\kcauto-kai.sikuli\combat.sikuli\combat.py", line 328, in _run_combat_logic
    self.lbas.assign_groups()
  File "E:\kcauto-kai\kcauto-kai.sikuli\combat.sikuli\lbas.py", line 49, in assign_groups
    node_obj = self.map.nodes[node]
KeyError: 'B'
```

**Comment by mrmin123** [comment]
The LBAS module already moves the panel out of the way when sorting. The
issue you are encountering is because node B is not defined in the map json
you are attempting to sortie to.

**Comment by sakura-quest** [comment]
Thanks for the info. I'll try to make a pull request to the winter 2018 event branch then with node B's definition.

**Comment by mrmin123** [comment]
Sure, could you branch off the `staging` branch and make  PR against `staging`? The event branch has already been merged into it as of 0.4.1.

**Comment by sakura-quest** [comment]
Yea sure. 
PR #111 has been made to staging.
